packages/browser/test/helpers.test.ts

```diff
@@ -56,16 +64,6 @@ describe('internal wrap()', () => {
     expect(wrap(wrapped)).toBe(wrapped);
   });
 
-  it('calls "before" function when invoking wrapped function', () => {
```

Comment:
l: why did we remove this test? (I guess it's redundant but just wanted to double-check)

Reply:
I removed the `before` option from the function, it was not used anymore :)